src/backends/quantumoptics/quantumoptics.jl

```diff
@@ -37,11 +37,13 @@ function observable(state::Union{<:Ket,<:Operator}, indices, operation)
     expect(op, state)
 end
 
-function project_traceout!(state::Union{Ket,Operator},stateindex,psis::Vector{<:Ket})
+function project_traceout!(state::Union{Ket,Operator},stateindex,psis::Union{Tuple,Vector{<:Ket}})
```

Comment:
let's use `Base.AbstractVecOrTuple` here (which is almost the same union), to keep things consistent